pkg/reconciler/v1alpha1/revision/revision.go

```diff
@@ -259,8 +259,16 @@ func (c *Reconciler) Reconcile(ctx context.Context, key string) error {
 	} else if err != nil {
 		return err
 	}
+
 	// Don't modify the informer's copy.
-	rev := original.DeepCopy()
+	rev, err := c.migrateConfigurationLabels(original.DeepCopy())
+
+	if err != nil {
+		logger.Warn("Failed to migrate revision labels", zap.Error(err))
```

Comment:
```suggestion
		logger.Warnw("Failed to migrate revision labels", zap.Error(err))
```

Reply:
I don't think you want `Warnw` since it seems to wants keys and values.

Reply:
We had an issue recently (last month) that basically if you don't use `Xw` methods the generated JSON won't be as nice.

Reply:
Obviously base logger does not support it, but all sugared one do.

Reply:
(this one is sugared, according to `pkg/logging/logging.go`)

Reply:
yeah I see it now..

```
Infow
{"level":"info","ts":1550085030.404629,"caller":"serving/main.go:17","msg":
"failed to fetch URL","error":"some error"}

Info
{"level":"info","ts":1550085030.404709,"caller":"serving/main.go:18","msg":
"failed to fetch URL{error 25 0  some error}"}
```